utils/fetchSwapAssets.ts

```diff
@@ -0,0 +1,23 @@
+import { Api } from "@cennznet/api";
+import { CENNZAsset } from "@/types";
+import fetchCENNZAssets from "@/utils/fetchCENNZAssets";
+
+/**
+ * Fetch and retuns array of CENNZnetAsset that are swapable
+ * We use `totalLiquidity` value to determine if token is swappable or not
+ *
+ * @param {Api} api
+ * @return {Promise<CENNZAsset[]>}
+ */
+export default async function fetchSwapAssets(api: Api): Promise<CENNZAsset[]> {
+	const assets = await fetchCENNZAssets(api);
+	const liquidityPredicates = await Promise.all(
+		assets.map(async ({ assetId, symbol }) => {
+			if (symbol === "CPAY") return true;
```

Comment:
can we check assetId here? cause there can be new asset with same symbol.. can't rely on symbol..
Better if we can get - api.query.cennzx.coreAssetId() and see if the assetId === coreAssetId

Reply:
good call, I have CPAY hard coded for now, addressed in https://github.com/cennznet/app-hub/pull/139

will probably find a way to use `api.query.cennzx.coreAssetId()` across the app